packages/api/src/controllers/room.ts

```diff
@@ -128,6 +128,16 @@ app.post(
       }
     }
 
+    const svc = new RoomServiceClient(
+      req.config.livekitHost,
+      req.config.livekitApiKey,
+      req.config.livekitSecret
+    );
+    const rooms = await svc.listRooms([req.params.roomId]);
+    if (rooms.length < 1) {
```

Comment:
Do we need to error if it's > 1 or safe to just ignore that case?

Reply:
Good point yeah it shouldn't be possible unless there was an issue on livekit's side, but makes sense to add a check to be made aware of it if it ever did happen.